qiskit/circuit/library/standard_gates/__init__.py

```diff
@@ -48,7 +48,25 @@
 
 def get_standard_gate_name_mapping():
     """Return a dictionary mapping the name of standard gates and instructions to an object for
-    that name."""
+    that name.
+
+    For Example:
+
+    .. code-block::
+
+        from qiskit.circuit.library.standard_gates import get_standard_gate_name_mapping
+
+        gate_name_map = get_standard_gate_name_mapping()
+        cx_object = gate_name_map["cx"]
+
+        print(cx_object)
+        print(type(cx_object))
+
+    .. parsed-literal::
+
+        Instruction(name='cx', num_qubits=2, num_clbits=0, params=[])
+        _SingletonCXGate"""
```

Comment:
1. Examples is a special keyword, as documented at https://sphinxcontrib-napoleon.readthedocs.io/en/latest/#docstring-sections. You can see an example with https://docs.quantum.ibm.com/api/qiskit/dev/assembler#disassemble of how it results in better formatting
2. Better to not use `.. parsed-literal`, even though we were using it a lot. See https://github.com/Qiskit/qiskit/pull/13257
3. You can now do `from qiskit.circuit.library.standard_gates import get_standard_gate_name_mapping` without needing `.standard_gates` 🚀
```suggestion
    Examples:

    .. code-block:: python

        from qiskit.circuit.library import get_standard_gate_name_mapping

        gate_name_map = get_standard_gate_name_mapping()
        cx_object = gate_name_map["cx"]

        print(cx_object)
        print(type(cx_object))

    .. code-block:: python

        Instruction(name='cx', num_qubits=2, num_clbits=0, params=[])
        _SingletonCXGate
    """
```